doc/submitter_guide.md

```diff
@@ -96,9 +95,7 @@ exit status.
 -----------------------------
 
 KCIDB accepts data in JSON, described by a versioned schema. The
-`kcidb-schema` tool will output the current schema version. However, all tools
-will accept data complying with older schema versions, as a rule. Any
-exceptions to that will be mentioned in KCIDB release notes. Pipe your data
+`kcidb-schema` tool will output the current schema version. However, all tools will accept data complying with older schema versions. Pipe your data
```

Comment:
Please also wrap this paragraph at 78 character boundary.

Reply:
I have combined all commits into one single commit with the changes you suggested above sir.